### ISSUE TITLE
Vulkan rendering broken since latest update

### ISSUE THREAD

**Issue by StingViperX** [Tsuey/L4D2-Community-Update#44]
Every time i try to launch Left 4 Dead 2 with -vulkan in launch options the game just crashes and a message saying "failed to create D3D device" appears.

**Comment by Treescrub** [comment]
Does this still occur?

**Comment by StingViperX** [comment]
Yes, the problem does still occur to this day. The problem seems to be linked with playing the game in fullscreen mode with -vulkan in launch options. 
The only way to play Left 4 Dead 2 with Vulkan is to play it in borderless fullscreen. I would love if the developers could fix this issue so i could play in fullscreen with Vulkan like i used to prior to an update back in August of 2021.
